templates/patterns.yml

```
@@ -0,0 +1,97 @@
+created_at: ${DATE}
+
+# Valid patterns to reject with explanation
+valid_patterns:
+  - id: console-log-lambda
+    pattern: "console.log"
+    condition:
+      files: ["**/lambda/**", "**/*-handler.js"]
+    reason: "We use console.log for CloudWatch logging in Lambda functions"
+    confidence: 1.0
+    auto_reply: |
+      Thanks for the feedback! We intentionally use console.log in our Lambda functions 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)